src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -84,6 +87,45 @@ public void run() {
         }
     }
 
+    private void niceSyncthing() {
+        new Thread() {
+            public void run() {
+                Process niceST = null;
+                DataOutputStream niceSTOut = null;
+                int ret = 1;
+                try {
+                    Thread.sleep(1000); // Wait a second before getting the Pid
+                    niceST = Runtime.getRuntime().exec("sh");
+                    niceSTOut = new DataOutputStream(niceST.getOutputStream());
+                    niceSTOut.writeBytes("set `ps |grep libsyncthing.so`\n");
+                    niceSTOut.writeBytes("ionice $2 be 7\n"); // best-effort, low priority
+                    niceSTOut.writeBytes("exit\n");
+                    log(niceST.getInputStream(), Log.INFO);
```

Comment:
Not sure if we need this as we also log exceptions etc. It also gets lumped in with syncthing logs. So I'd use a different tag or just don't log the output.